sql/driver/conn.go

```diff
@@ -29,6 +29,7 @@ import (
 type conn struct {
 	sender  Sender
 	session []byte
+	txn     []byte
```

Comment:
These actually correspond to what we have in the protobuf. We could change the protobuf if you like.
On a related note storing the session with the connection also doesn't work out the way you might expect. I trying setting the database in the bank example and it looks rather awkward. It was more intuitive to have the settings be part of the database. That's for another change.